bun-tests/fake-snippets-api/routes/orders/order-with-circuit-json-or-package-release.test.ts

```diff
@@ -0,0 +1,80 @@
+import { getTestServer } from "bun-tests/fake-snippets-api/fixtures/get-test-server"
+import { test, expect } from "bun:test"
+
+const setupTestServer = async (
+  withCircuitJson: boolean,
+  withPackageRelease: boolean,
+) => {
+  return await getTestServer()
+}
+
+test("create order with only circuit_json (✅)", async () => {
+  const {
+    axios,
+    seed: { order },
+  } = await setupTestServer(true, false)
+
+  const response = await axios.post("/api/order_quotes/create", {
+    vendor_name: "jlcpcb",
+    circuit_json: order.circuit_json,
```

Comment:
don't use that seed data here, it's confusing because order_quotes are created BEFORE orders, you can just use e.g. an empty array (or even better, with a single source component or something)